__tests__/confirm_API_data_browse.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test, expect } from '@playwright/test';
+
+test('API Datasource Browsing', async ({ page }) => {
+  await page.goto('/');
+  await page.locator('[id="\\(API\\)GNURadioSigMFRepo"]').click();
+  await page.getByRole('link', { name: 'bluetooth' }).click();
+  await expect(page.getByText('Zoom Level')).toBeVisible();
+});
```

Comment:
This test it is making the assumption that the user does have the database populated with this entry on it.